torch/csrc/inductor/aoti_runtime/device_utils.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_runtime/cpu_utils.h>
```

Comment:
The exclusiveness of the sets of macros is not obvious from this file. May be better to just keep this file and merge the contents of the two files into this one.